apps/sms/views/shared/test/unit/utils_test.js

```diff
@@ -1153,6 +1153,157 @@ suite('Utils', function() {
     });
   });
 
+  suite('Utils.throttle', function() {
+    setup(function() {
+      this.sinon.useFakeTimers(1);
+    });
+
+    test(
+      'right typeof parameters', 
+      function() {
+        assert.throw(
+          () => Utils.throttle(null), 
+          Error, 
+          'func must be a Function');
```

Comment:
nit: please put the close parenthesis to new line just like the close bracket here and below.